test/server/graphql/v1/collective.test.js

```diff
@@ -1294,4 +1302,53 @@ describe('server/graphql/v1/collective', () => {
       expect(cacheDelSpy.callCount).to.equal(0);
     });
   });
+
+  describe('contributors', () => {
+    const contributorsQuery = gql`
+      query Collective($slug: String!) {
+        Collective(slug: $slug) {
+          id
+          contributors {
+            id
+            roles
+          }
+          admins: contributors(roles: ADMIN) {
+            id
+          }
+          tiers {
+            contributors {
+              id
+            }
+          }
```

Comment:
Here we previously had 4 queries to `getContributorsForCollective`: one for the contributors, one for the admins, and one for each of the two tiers.